alerting/src/main/kotlin/org/opensearch/alerting/AlertingPlugin.kt

```diff
@@ -141,10 +141,14 @@ internal class AlertingPlugin : PainlessExtension, ActionPlugin, ScriptPlugin, R
     companion object {
         @JvmField val OPEN_SEARCH_DASHBOARDS_USER_AGENT = "OpenSearch-Dashboards"
         @JvmField val UI_METADATA_EXCLUDE = arrayOf("monitor.${Monitor.UI_METADATA_FIELD}")
-        @JvmField val MONITOR_BASE_URI = "/_opendistro/_alerting/monitors"
-        @JvmField val DESTINATION_BASE_URI = "/_opendistro/_alerting/destinations"
+        @JvmField val MONITOR_BASE_URI = "/_plugins/_alerting/monitors"
+        @JvmField val DESTINATION_BASE_URI = "/_plugins/_alerting/destinations"
+        @JvmField val LEGACY_MONITOR_BASE_URI = "/_opendistro/_alerting/monitors"
```

Comment:
These should probably be `LEGACY_OPENDISTRO_...`, in the future we will legacy something else.

Reply:
Updated in the same PR.